satellite/payments/stripepayments/stripe.go

```diff
@@ -123,6 +123,10 @@ func (s *service) AddPaymentMethod(ctx context.Context, params payments.AddPayme
 		},
 	}, nil
 }
+func extractStripeErrorMessage(inErr error) error {
+	stripeErr := inErr.(*stripe.Error)
```

Comment:
If it isn't `stripe` err it will panic
https://stripe.com/docs/api/errors/handling